### PR TITLE
GSoC2022: Added OWASP CRS and 6 new ideas

### PR DESCRIPTION
**Pull request by dune73**

### CODE REVIEW

File: pages/initiatives/gsoc/gsoc2022ideas.md [hunk 1]
```diff
@@ -455,3 +455,127 @@ SecRule REQUEST_URI "^/api/graphql$" "ctl:requestBodyProcessor=GRAPHQL"
...
+
+##### Mentors
+
+The CRS team will assign a mentor to contributors. In the meantime, the following two CRS project leaders will be your contacts:
+
+* [Christian Folini](mailto:christian.folini@owasp.org)
+* [Felipe Zipitría](mailto:felipe.zipitria@owasp.org)
+
+
+#### CRS Idea 5: New plugin for &lt;enter-your-cool-idea&gt;
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2022ideas.md:545]:
```suggestion
* [Felipe Zipitría](mailto:felipe.zipitria@owasp.org)

#### CRS Idea 5: New plugin for &lt;enter-your-cool-idea&gt;
```

File: pages/initiatives/gsoc/gsoc2022ideas.md [hunk 2]
```diff
@@ -455,3 +455,127 @@ SecRule REQUEST_URI "^/api/graphql$" "ctl:requestBodyProcessor=GRAPHQL"
...
+
+![Preferred for "Small" GSoC 2022 project](https://img.shields.io/badge/small%20size%20(~100h)-preferred-green)
+![Possible for "Medium" GSoC 2022 project](https://img.shields.io/badge/medium%20size%20(~175h)-possible-yellow)
+
+##### Description
+
+We have recently added plugin functionality to CRS. The [Plugin Registry](https://github.com/coreruleset/plugin-registry) has a decent overview over existing plugins and this [blog post](https://coreruleset.org/20220209/introducing-the-fake-bot-plugin/) does a good job describing a very cool plugin for inspiration.
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2022ideas.md:552]:
```suggestion
We have recently added plugin functionality to CRS. The [Plugin Registry](https://github.com/coreruleset/plugin-registry) has a decent overview of existing plugins and this [blog post](https://coreruleset.org/20220209/introducing-the-fake-bot-plugin/) does a good job describing a very cool plugin for inspiration.
```

File: pages/initiatives/gsoc/gsoc2022ideas.md [hunk 3]
```diff
@@ -455,3 +455,127 @@ SecRule REQUEST_URI "^/api/graphql$" "ctl:requestBodyProcessor=GRAPHQL"
...
+
+##### Mentors
+
+The CRS team will assign a mentor to contributors. In the meantime, the following two CRS project leaders will be your contacts:
+
+* [Christian Folini](mailto:christian.folini@owasp.org)
+* [Felipe Zipitría](mailto:felipe.zipitria@owasp.org)
+
+
+#### CRS Idea 6: Setup automatic workflow to scan existing CVE PoC repository and run PoCs against sandbox
```
Comment (kingthorin) [inline pages/initiatives/gsoc/gsoc2022ideas.md:564]:
```suggestion
* [Felipe Zipitría](mailto:felipe.zipitria@owasp.org)

#### CRS Idea 6: Setup automatic workflow to scan existing CVE PoC repository and run PoCs against sandbox
```

**Comment by dune73** [comment]
Thanks @kingthorin. I think I adopted all your suggestions.

**Review by kingthorin — APPROVED** [review]

**Comment by dune73** [comment]
Thank you! This was swift.

**Comment by kingthorin** [comment]
No problem, happy to help.
Luckily you submitted while I was working on some other stuff :wink:

**Pull request merged**